code/modules/antagonists/_common/antag_datum.dm

```diff
@@ -82,6 +113,7 @@ GLOBAL_LIST_EMPTY(antagonists)
 		owner.current.key = C.key
 
 /datum/antagonist/proc/on_removal()
+	assigned_targets.Cut()
 	remove_innate_effects()
```

Comment:
Is wiping the list neccesary here? Since it's being deleted anyway a few lines later.

Reply:
The cut is needed for a proper GC. But I'll move it to the `Destroy` which is a better place for that